arch/arm/soc/nxp_kinetis/kwx/Kconfig.soc

```diff
@@ -43,14 +59,58 @@ config SOC_PART_NUMBER_MKW41Z256VHT4
 config SOC_PART_NUMBER_MKW41Z512VHT4
 	bool
 
+config SOC_PART_NUMBER_MKW22D512VHA5
```

Comment:
nit: Could you please move these new lines above the KW40 so they're in numerical order?